mediathread/templates/lti_auth/landing_page.html

```diff
@@ -45,6 +45,7 @@
                 <p class="lead text-center">
                     <a class="btn btn-primary btn-lg" href="{{landing_url}}" target="_blank">Launch Mediathread</a>
                 </p>
+                <iframe width="560" height="315" src="https://www.youtube.com/embed/xwy3I1AMLH4" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Could the video be centered? I think the easiest way to accomplish this is to wrap the iframe in a div. Something like `<div class="text-center w-100">`.